app/Anime.php

```diff
@@ -259,12 +259,13 @@ public function typesenseQueryBy(): array
             'title_english_transformed',
             'title_japanese',
             'title_japanese_transformed',
+            'title_synonyms',
```

Comment:
@pushrbx We removed synonyms because they were degrading results. Should we be adding it back?

Reply:
Yes. While testing with a wider range of people we found that this improves the hit results.
The most noteable edge case is the situation where the datasheet on mal doesn't have an english title in gray under the original title, so the parser just falls back to the original title so the english title will be same as the original one, however the actual english title appears as a synonym/alternative title. You can observe this with `Utawarerumono`, you can't search for its english title `The One Being Sung`.
Similar situation for `Slayers`: if the community decides to add localised titles for it e.g. the hungarian `Kis boszorkány` then people using the api from that country would want to be able to search for the localised title.
Worth noting with this change I might have broke something else. Or last time we didn't have the correct weights for ranking.

Reply:
Fair enough.

> Worth noting with this change I might have broke something else. Or last time we didn't have the correct weights for ranking.

Could be, we'll have to keep an eye on it. I don't remember what was diminishing the results last time with synonyms. Could've been incorrect weights too.

Staging currently has this change so I'll test it out further there.